oioioi/sinolpack/package.py

```diff
@@ -1248,7 +1248,7 @@ def _get_model_solutions_sources(self):
         """
         lang_exts_list = getattr(settings, 'SUBMITTABLE_EXTENSIONS', {}).values()
         extensions = [ext for lang_exts in lang_exts_list for ext in lang_exts]
-        regex = r'^%s[0-9]*([bs]?)[0-9]*(sol.*)?\.(' + '|'.join(extensions) + ')'
+        regex = r'^%s[0-9]*([bs]?)[0-9]*(_.*)?\.(' + '|'.join(extensions) + ')'
```

Comment:
Personally I prefer the previous regex, at this moment other utility programs like inwer/ingen/chk are single character apart to match new regex.